server.js

```diff
@@ -1,17 +1,20 @@
 var Discord = require("discord.js");
 var bot = new Discord.Client();
-var mtrAggregator = new require("./aggregators/rules/mtr.js");
-var ipgAggregator = new require("./aggregators/rules/ipg.js");
-var crAggregator = new require("./aggregators/rules/cr.js");
-var jarAggregator = new require("./aggregators/rules/jar.js");
-var cardAggregator = new require("./aggregators/mtg_cards/cards.js");
 
-const mtr = "!mtr";
-const ipg = "!ipg";
-const cr = "!cr";
-const jar = "!jar";
-const card = "!card";
-const help = "!help";
+var mtrAggregator = new (require("./aggregators/rules/mtr.js"))();
+var ipgAggregator = new (require("./aggregators/rules/ipg.js"))();
+var crAggregator = new (require("./aggregators/rules/cr.js"))();
+var jarAggregator = new (require("./aggregators/rules/jar.js"))();
+var cardAggregator = new (require("./aggregators/mtg_cards/cards.js"))();
+
+var handlers = {
+	"!mtr": mtrAggregator.getContent,
+	"!ipg": ipgAggregator.getContent,
+	"!cr": crAggregator.getContent,
+	"!define": crAggregator.getGlossaryEntry,
```

Comment:
Not a big fan of having 2 different entry points for `crAggregator` - I'd rather prefer passing in the command argument along with the parameter and let `crAggregator` decide for itself how to handle a `!cr` versus a `!define`.